src/spec-node/singleContainer.ts

```diff
@@ -174,7 +174,11 @@ async function buildAndExtendImage(buildParams: DockerResolverParameters, config
 		if (buildParams.buildxPush) {
 			args.push('--push');
 		} else {
-			args.push('--load'); // (short for --output=docker, i.e. load into normal 'docker images' collection)
+			if (buildParams.buildxOutputType) { 
+				args.push('--output', `type=${buildParams.buildxOutputType},dest=${buildParams.buildxOutputDest}`);
```

Comment:
buildx seems to have a few variations on what options an output type can take: https://docs.docker.com/engine/reference/commandline/buildx_build/#output

Should we have just a single option that is passed directly to buildx's `--output` to cover all cases? This would also be useful if/when buildx adds more options to `--output`.

Reply:
I'm fine with doing that, wasn't sure which way was more in the spirit of the current option structure. Def a case to be made that a transparent `--output` would be less upkeep as Docker changes. Will do it.